ui/views/channels.go

```diff
@@ -538,6 +538,58 @@ func NewChannels(cfg *config.View, chans *models.Channels) *Channels {
 					return color.White(opts...)(printer.Sprintf("%8d", c.UpdatesCount))
 				},
 			}
+		case "BASE OUT":
```

Comment:
White space makes it hard to distinguish columns.
Can you change it for `BASE_OUT`, `RATE_OUT`, `BASE_IN`, `RATE_IN` ?

Reply:
I agree, should we change the "LAST UPDATE" column as well?

Reply:
I think it will be changed but not in this PR, thank you !